client/app/about.tsx

```diff
@@ -0,0 +1,227 @@
+      "https://thumbs.dreamstime.com/b/close-up-happy-teenage-girl-1465128.jpg",
+      linkedin: 'linkedin:  linkedin/sadasdasasdfagsdjhadvjsd',
+  },
+  {
+    name: "Kseniya",
+    id: "7",
+    about:
+      "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat velit alias sit earum porro modi eligendi neque accusantium tenetur ex rerum architecto eum cumque, explicabo officia nihil dicta nostrum amet.",
+    image:
+      "https://www.shutterstock.com/shutterstock/photos/623101280/display_1500/stock-photo-portrait-of-serious-business-woman-standing-outside-building-623101280.jpg",
+    linkedin: 'linkedin: linkedin/sadasdhayGHEOQDWJLNRLIWEEasd',
+  },
+  {
+    name: "Tomer cohen",
+    id: "8",
+    about:
+      "Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat velit alias sit earum porro modi eligendi neque accusantium tenetur ex rerum architecto eum cumque, explicabo officia nihil dicta nostrum amet.",
+    image:
+      "https://media.gettyimages.com/id/571751597/photo/portrait-of-young-business-woman-holding-paper-work-looking-away-side-view.jpg?s=612x612&w=gi&k=20&c=oMhPP2YzYQVx42_gCMGPm1R_xZE5yWLlqcSl7diBYIM=",
+      linkedin: 'linkedin:  linkedin/sadasJASDGIWAHRFKUQWHLQoilasdaidasd',
+  },
+];
+
+const imageBackground = {
+  uri: "https://www.teahub.io/photos/full/1-16896_3d-world-map-hd-wallpaper-best-of-map.jpg",
+};
+const About = () => {
+  const [expandedUser, setExpandedUser] = useState<string | null>(null);
+  const [expandedManager, setExpandedManager] = useState<string | null>(null);
+
+  const toggleUser = (userId: string) => {
+    if (expandedUser === userId) {
+      setExpandedUser(null);
+    } else {
+      setExpandedUser(userId);
+    }
+  };
+
+  const toggleManager = (managerId: string) => {
+    if (expandedManager === managerId) {
+      setExpandedManager(null);
+    } else {
+      setExpandedManager(managerId);
+    }
+  };
+
+  return (
+    <View style={styles.box}>
+      <ImageBackground source={imageBackground} style={styles.imageBackground}>
+        <ScrollView style={styles.container}>
+      <Text style={{fontSize:50,  color: "white"}}>About as:</Text>
+          {managers.map((manager) => (
+            <View key={manager.id} style={styles.listItem}>
+              <View style={styles.checkboxContainer}>
+                <Switch
+                  value={expandedManager === manager.id}
+                  onValueChange={() => toggleManager(manager.id)}
+                />
+                <Image style={styles.image} source={{ uri: manager.image }} />
+                <Text style={styles.text}>{manager.name}</Text>
+              </View>
+
+              {expandedManager === manager.id && (
+                <View style={styles.subList}>
+                  <Text style={styles.text}>{manager.about}</Text>
+                  <Text style={{ color: "green" }}>{manager.linkedin}</Text>
+                </View>
+              )}
+            </View>
+          ))}
```

Comment:
Why can't the manager be in the same loop as the users? there's not difference in functionality between the two.

Reply:
fix